modules/developers-guide/pages/cli-reference/dfx-stop.adoc

```diff
@@ -35,7 +35,7 @@ You can stop the {IC} network processes that are running in the background by ch
 dfx stop
 ----
 
-If the local {IC} network is running in a current shell rather than in the background, open a new terminal shell, change to a project directory, then run the `+dfx stop+` command.
+If the local {IC} replica is running in a current shell rather than in the background, open a new terminal shell, change to a project directory, then run the `+dfx stop+` command.
```

Comment:
```suggestion
If the local canister execution environment is running in a current shell rather than in the background, open a new terminal shell, change to a project directory, then run the `+dfx stop+` command.
```